src/include/OpenImageIO/filesystem.h

```diff
@@ -193,52 +210,16 @@ OIIO_API FILE *fopen (string_view path, string_view mode);
 ///
 OIIO_API std::string current_path ();
 
-/// Deprecated: Will not work correctly on GCC with MingW, prefer
-/// the version below taking a ofstream**
 /// Version of std::ifstream.open that can handle UTF-8 paths
 ///
-OIIO_API void open (std::ifstream &stream, string_view path,
+OIIO_API void open (OIIO_NAMESPACE::ifstream &stream, string_view path,
```

Comment:
Just OIIO::ifstream.

(They are equivalent, but let's stick to the namespace alias we use everywhere else.)